### PR TITLE
Doc: Tips on how to list available JVMs using coursier

### PR DESCRIPTION
**Pull request by jatcwang**

### CODE REVIEW

File: website/docs/reference/cli-options.md [hunk 1]
```diff
@@ -639,7 +639,7 @@ Set the Java home directory
 
 Aliases: `-j`
 
-Use a specific JVM, such as `14`, `adopt:11`, or `graalvm:21`, or `system`
+Use a specific JVM, such as `14`, `adopt:11`, or `graalvm:21`, or `system`. scala-cli uses **cousier** to fetch JVMs, so you can use `cs java --available` to list the available JVMs.
```
Comment (tgodzik) [inline website/docs/reference/cli-options.md]:
```suggestion
Use a specific JVM, such as `14`, `adopt:11`, or `graalvm:21`, or `system`. scala-cli uses [coursier](https://get-coursier.io/) to fetch JVMs, so you can use `cs java --available` to list the available JVMs.
```

**Review by tgodzik — CHANGES_REQUESTED** [review]
Thanks! It might useful to link to it and there is a small typo.

File: website/docs/reference/cli-options.md (hunk 1 above)
Comment (jatcwang) [reply website/docs/reference/cli-options.md]: Updated thanks!

**Comment by tgodzik** [comment]
Could you run `./mill -i 'generate-reference-doc[]'.run` ? We have a special reference doc for SIP purposes and it needs to be up to date. Thanks!

**Review by tgodzik — APPROVED** [review]
Thanks!

**Pull request merged**